juicefs-1.3.yaml

```diff
@@ -0,0 +1,139 @@
+package:
+  name: juicefs-1.3
+  version: "1.3.0"
+  epoch: 0
+  description: JuiceFS is a distributed POSIX file system built on top of Redis and S3.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - fuse3
+    provides:
+      - juicefs=${{package.full-version}}
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/juicedata/juicefs
+      tag: v${{package.version}}
+      expected-commit: 30190ca1094d26e85f19a979ca51b0ea19af1eaa
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/net@v0.38.0
+        github.com/golang/glog@v1.2.4
+        github.com/tidwall/gjson@v1.9.3
+        golang.org/x/oauth2@v0.27.0
+
+  - uses: go/build
+    with:
+      packages: .
```

Comment:
Why we don't pass `tags`? https://github.com/juicedata/juicefs/blob/fcef17452db35ca8875ce1475baae37340879d42/Makefile#L50

If there is a reason, can we drop a comment here?

Reply:
After adding the tags, we need `ceph`, `glusterfs` and `foundationdb` dev packages in the build environment. For now, we're just going for building the main `juicefs` binary.
https://github.com/juicedata/juicefs/blob/fcef17452db35ca8875ce1475baae37340879d42/Makefile#L28